src/content/docs/kubernetes-pixie/kubernetes-integration/get-started/changes-since-v3.mdx

```diff
@@ -170,8 +165,8 @@ The following agent options were previously "aliased" in the root of the `values
 
 * `logFile` has been replaced by `common.agentConfig.log_file`.
 * `eventQueueDepth` has been replaced by `common.agentConfig.event_queue_depth`.
-* `customAttributes` has changed in format to a yaml object. The previous format, a manually json-encoded string e.g. `{"team": "devops"}`, is still accepted although discouraged.
-* Previously, `customAttributes` had a default `clusterName` entry which might have unwanted consequences if removed. This is no longer the case, users may now safely override `customAttributes` on its entirety.
+* `customAttributes` has changed in format to a yaml object. The previous format, a manually json-encoded string e.g. `{"team": "devops"}` is deprecated.
+* Previously, `customAttributes` had a default `clusterName` entry which might have unwanted consequences if removed. This is no longer the case, users may now safely override `customAttributes` on its entirety. We will add `clusterName` for you while it is needed.
```

Comment:
"while" or "if" (or when?)

Reply:
I am removing this phrase because it is unnecessary information for the user.

Reply:
Solved in the commit [#5272112](https://github.com/newrelic/docs-website/pull/7322/commits/52721120b118b37ff91856396a750b4964b520c4)